ui/src/components/common/SummaryPageLayout/index.tsx

```diff
@@ -4,16 +4,103 @@ import chevronLeft from "../../../images/chevron-m-left.png";
 import chevronRight from "../../../images/chevron-m-right.png";
 
 import "./style.css";
+import {
```

Comment:
please move css files to the end of import

Reply:
Updated